pint-cli/src/build.rs

```diff
@@ -102,6 +111,15 @@ pub(crate) fn cmd(args: Args) -> anyhow::Result<()> {
 
     // Build the given compilation plan.
     let mut builder = pint_pkg::build::build_plan(&plan);
+    let options = pint_pkg::build::BuildOptions {
+        salt: args.salt.unwrap_or_default(),
```

Comment:
It looks like this may end up applying the salt to every package, but we only want to apply it to the "top-level" contract package that was targeted for building.

Rather than making this a single salt, one option could be to make this something liike `HashMap<Manifest, Salt>`, so that each package can check whether or not there was a custom salt specified for it during the build?

Reply:
Alright, I addressed this in https://github.com/essential-contributions/pint/pull/965/commits/63648b9dabf1c65bf1f97c4060ef578b4bf08fa7.. Please let me know if this makes sense!